modules/gapi/test/own/mat_tests.cpp

```diff
@@ -384,4 +384,23 @@ TEST(OwnMat, ROIView)
     << to_ocv(roi_view) << std::endl
     << expected_cv_mat  << std::endl;
 }
+
+TEST(OwnMat, CreateWithNegativeDims)
+{
+    Mat own_mat_t1, own_mat_t2, own_mat_t3;
+    EXPECT_ANY_THROW(own_mat_t1.create(cv::Size{-1, -1}, CV_8U));
+    EXPECT_ANY_THROW(own_mat_t2.create(cv::Size{ 1, -1}, CV_8U));
+    EXPECT_ANY_THROW(own_mat_t3.create(cv::Size{-1,  1}, CV_8U));
+}
+
+TEST(OwnMat, ExistCreateWithNegDims)
+{
+    Mat own_mat_t1(-1, -1, CV_8U, nullptr);
```

Comment:
Is it really necessary to pass `nullptr` here ?

Reply:
own::Mat's constructor is `Mat(int, int, int, void* _data, size_t)`. I should put something in `_data`.

Reply:
Ok